src/openseadragon.js

```diff
@@ -840,6 +840,23 @@ if (typeof define === 'function' && define.amd) {
     }());
 
     /**
+     * Test whether the submitted canvas is tainted or not.
+     * @argument {Canvas} canvas The canvas to test.
+     * @returns {Boolean} True if the canvas is tainted.
+     */
+    $.isCanvasTainted = function(canvas) {
```

Comment:
I suppose now that this is here we can get rid of the copy in the tests: 

https://github.com/openseadragon/openseadragon/blob/master/test/modules/basic.js#L322